### ISSUE TITLE
Add option to show whitespace characters in diff

### ISSUE THREAD

**Issue by andresovela** [extrawurst/gitui#1147]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to see whitespace characters in the diff panel. Sometimes I see stuff like this:

![Screenshot from 2022-02-24 10-40-04](https://user-images.githubusercontent.com/8990961/155498923-0ff40758-8ce9-44c6-8a2a-9bfdc9ffdf42.png)

My IDE displays the whitespace characters like this:

![Screenshot from 2022-02-24 10-41-04](https://user-images.githubusercontent.com/8990961/155499059-17314ba8-4bb4-456f-bc25-05056c0c5453.png)

**Describe the solution you'd like**
An option to enable displaying whitespace characters.

**Describe alternatives you've considered**
Maybe highlighting the parts of the line that changed could help.

**Comment by extrawurst** [comment]
this makes sense eventually but is not high on my priority list since I want/need to rewrite the whole diffing from scratch eventually to support external diff crates and not using the libgit2 internal one

**Comment by Kamilcuk** [comment]
@extrawurst Hi, truly an amazing project, and thank you for making it. I can click `.` for more options, type `o` for options, and then navigate to "Ignore whitespace" and change to true.

This works (!) thank you. But it only works for "Status [1]" view. But it does not work for "Log [2]" view after navigating to a commit.

I tried following https://github.com/extrawurst/gitui/blob/master/src/tabs/revlog.rs to find missing link of options. I see at https://github.com/extrawurst/gitui/blob/289eddd9beeffeb57ca3ae9f55380dd05996c23a/asyncgit/src/sync/commit_files.rs#L24 the options are not passed, but I do not know about the codebase.

**Comment by extrawurst** [comment]
> I can click `.` for more options, type `o` for options, and then navigate to "Ignore whitespace" and change to true.
> 
> This works (!) thank you. But it only works for "Status [1]" view. But it does not work for "Log [2]" view after navigating to a commit.

good catch but an entirely different topic. please feel free to create another issue (maybe we have one for it already?)

**Comment by Kamilcuk** [comment]
Och, this issue is to _show_ and not ignore the whitespaces. My bad, sorry.
